<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="importLoops">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Loops</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteLoop">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Loop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportMaster">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>114</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Master</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replace">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>114</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectMaster">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectLoop1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectLoop2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>114</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Slice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftArrow">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightArrow">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QDial" name="sensKnob1">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="sensKnob2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>205</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="sensKnob3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>255</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>541</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>â–¶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-color: rgb(66, 69, 183);
border-width: 3px;        
border-style: solid;
border-radius: 25px;

</string>
    </property>
    <property name="text">
     <string>LoopStorm</string>
    </property>
   </widget>
   <widget class="QLabel" name="loop0">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>520</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Master Loop</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="loop1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>520</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Loop 1</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="loop2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>520</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Loop 2</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
